src/pages/components/menu-buttons/style.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Menu buttons
+tabs: ['Usage', 'Style', 'Code', 'Accessibility']
+---
+
+## Color
+
+Enabled and interactive states of menu buttons follow the
+[button style](https://carbondesignsystem.com/components/button/style)
+guidelines. Menu buttons can use primary, tertiary, or ghost buttons. Combo
+buttons can only use a primary button. Overflow menu can only use a ghost
+button.
+
+The menu used in menu buttons should follow the [menu style].
+
+### Menu button
+
+<Row>
+<Column colLg={8}>
+
+![Menu button variants](images/style-menu-buttons.png)
+
+</Column>
+</Row>
+
+<Caption>Example of menu button using three different types of buttons</Caption>
+
+### Combo button
+
+<Row>
+<Column colLg={8}>
+
+![Combo button](images/style-combo-button.png)
+
+</Column>
+</Row>
+
+<Caption>Example of a combo button using a primary button</Caption>
+
+### Overflow menu
+
+<Row>
+<Column colLg={8}>
+
+![Overflow menu](images/style-overflow-menu.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Overflow menu appears as a ghost button when closed and once it opens the
+  color changes to the correct $layer color token to match the menu.
+</Caption>
+
+## Typography
+
+| Element          | Font-size (px/rem) | Font-weight   | Type token         |
+| ---------------- | ------------------ | ------------- | ------------------ |
+| Button text      | 14 / 0.875         | Regular / 400 | `$body-compact-01` |
+| Menu option text | 14 / 0.875         | Regular / 400 | `$body-compact-01` |
+
+## Structure
+
+The structure of menu buttons should follow the
+[button style](https://carbondesignsystem.com/components/button/style)
+guidelines.
+
+### Menu button
+
+<div className="image--fixed">
+
+![Menu button structure](images/style-structure-menu-button.png)
+
+</div>
+
+<Caption>
+  Menu button in default width (left) and menu button in fluid width (right)
+</Caption>
+
+### Combo button
+
+<div className="image--fixed">
+
+![Combo button structure](images/style-structure-combo-button.png)
```

Comment:
These structure images appear correctly. I would try to do the same thing with the Menu style tab structure images to fix them.

Reply:
I am going to look into this issue more.